addons/wakatime/api_key_modal.gd

```diff
@@ -1,14 +1,14 @@
 @tool
 extends PopupPanel
 
+var wakatime_ref = null
+var settings = null
+
```

Comment:
I use the linter from gdtoolkit and it complains if the vars are not declared above the onready vars.